server/src/controllers/course.ts

```diff
@@ -157,6 +281,14 @@ const courseInstanceAddRequestSchema: yup.AnyObjectSchema = yup.object().shape({
   responsibleTeacher: yup
     .number()
     .required(),
+  minCredits: yup
+    .number()
+    .min(0)
```

Comment:
Should this be 1 instead? Is there a course that can give zero credits?

Reply:
I was also thinking about this and did some Sisu research. There exists courses with 0 credits like the bachelors 'Maturity Test (0 cr)'. Though I am not sure if that kind of course would ever end up in this system? Nevertheless I think it's better to have the option for 0 credits.

Reply:
If the possibility for a 0 credit course exists then I agree that it's better to keep this as 0.